tests/chainermn_tests/communicator_tests/test_communicator.py

```diff
@@ -899,3 +908,14 @@ def test_send_recv_obj(self):
         self.check_send_recv_obj(5, tag=5, use_any_recv=False, use_status=True)
 
         self.teardown()
+
+    def test_config(self):
```

Comment:
How about adding a test in which `with' statement is used?
`with` statement can be used in the new config system.